### ISSUE TITLE
Hydration node mismatch / Multiple versions of Lit warnings

### ISSUE THREAD

**Issue by fernandofranca** [prashantpalikhe/nuxt-ssr-lit#73]
I set up a small sample project using Nuxt@3.0.0 nuxt-ssr-lit@1.5.1-beta.2.

In the [project](https://github.com/fernandofranca/nuxt-3-lit-ssr) I added a simple Lit component and a Nuxt page using that component.

When I start the Nuxt in development mode I get two warnings:

> Multiple versions of Lit loaded. Loading multiple versions is not recommended. See https://lit.dev/msg/multiple-versions for more information.

> [Vue warn]: Hydration node mismatch:
> - Client vnode: div 
> - Server rendered DOM: <my-element>​…​</my-element>​  
>   at <LitWrapper.client> 
>   at <App key=1 > 
>   at <NuxtRoot>

I'm not sure why it is reporting multiple versions of Lit, as when I run `yarn why lit` I only see one version of Lit being used.

```
$ yarn why lit
├─ @lit-labs/ssr-client@npm:1.1.1
│  └─ lit@npm:2.7.3 (via npm:^2.0.0)
│
├─ @lit-labs/ssr@npm:3.1.2
│  └─ lit@npm:2.7.3 (via npm:^2.7.0)
│
└─ nuxt-app@workspace:.
   └─ lit@npm:2.7.3 (via npm:2.7.3)
```

If I remove the Lit elements from the template, the "Hydration node mismatch warning" goes away. When I remove the import for the custom component, the "Multiple versions of Lit" warning also goes away.

**To Reproduce**
1. Clone the reproduction repo: https://github.com/fernandofranca/nuxt-3-lit-ssr
2. Install with `yarn`
3. Start the dev server with `yarn dev`
4. Open the browser console to find the warning messages

**Expected behavior**
No warning messages.

 - nuxt: 3.0.0
 - lit: 2.7.3
 - nuxt-ssr-lit: 1.5.1-beta.2
 - Browser: Chrome 111

**Additional context**

**Comment by prashantpalikhe** [comment]
Hi @fernandofranca 

It seems that a prelease version was being installed. I have made v1.5.1 available which should remove the Hydration mismatch issue. 

Can you try uninstalling and installing the module again?

Regarding the multiple version of Lit, on dev mode, if you inspect `litHtmlVersions` in the browser console, you will see 

<img width="215" alt="image" src="https://user-images.githubusercontent.com/2657709/235051086-f93e56f8-87f1-497f-94b7-22a7eff68165.png">

This causes the multiple Lit version issue.  But this is stripped off in the production build. 

We can look into it why exactly this occurs in dev mode. But I have not done that yet since it is harmless and is not ending up in the prod. build.

**Comment by fernandofranca** [comment]
Thanks for the fast response @prashantpalikhe 🙇‍♂️!

I can confirm version 1.5.1 solves the "Hydration node mismatch" warning 🎉. 

Regarding the "Multiple versions of Lit" warning it is still there, hopefully is not a big deal.

`litHtmlVersions` was showing two different versions `['2.7.2', '2.7.3']`. I set yarn resolution to load only 2.7.3, but it still warns and now shows that same version twice, as in your screenshot.

**Comment by prashantpalikhe** [comment]
@fernandofranca Yeah, in prod mode, confirm that you only have one version being loaded. Should be good. 

If you confirm that, I can go ahead and close this issue :)
